pkgs/swift2objc/lib/src/generate_wrapper.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:io';
+import '../swift2objc.dart';
+import 'config.dart';
+import 'generator/generator.dart';
+import 'parser/parser.dart';
+import 'transformer/transform.dart';
+
+Future<void> generateWrapper(Config config) async {
+  final parentDir = config.tempDir ?? defaultTempDir;
+  final tempDir = await parentDir.createTemp(defaultTempDirPrefix);
+
+  await _generateSymbolgraphJson(
+    config.inputFiles,
+    tempDir,
+  );
+
+  String? symbolgraphJsonPath;
+  await for (final entity in tempDir.list()) {
+    if (entity is! File) continue;
+    if (entity.path.endsWith(symbolgraphFileSuffix)) {
+      symbolgraphJsonPath = entity.path;
+      break;
+    }
+  }
+
+  if (symbolgraphJsonPath == null) {
+    throw Exception('Could not find generated symbolgraph json file.');
+  }
+
+  final declarations = parseAst(symbolgraphJsonPath);
+  final transformedDeclarations = transform(declarations);
+  final wrapperCode = generate(transformedDeclarations);
+
+  await File(config.outputFile).writeAsString(wrapperCode);
+
+  if (config.deleteTempAfterDone) {
+    await for (final entity in tempDir.list()) {
```

Comment:
You don't need to do this loop. You can just do `tempDir.delete(recursive: true)`

Reply:
done